packages/verbs-sdk/src/adapters/privy.ts

```diff
@@ -0,0 +1,59 @@
+import { PrivyClient } from '@privy-io/server-auth'
+import type { Address } from 'viem'
+
+import type { WalletProvider } from '../types/wallet.js'
+import { Wallet } from '../wallet.js'
+
+/**
+ * Privy wallet provider implementation
+ * @description Wallet provider implementation using Privy service
+ */
+export class PrivyWalletProvider implements WalletProvider {
+  private privy: PrivyClient
+  private chainId: number = 10 // TODO: make configurable
```

Comment:
agree eventually we want this to be configurable but in the meantime we may want to set this to 130 as its Unichain's chainId and the pool we're going to demo is there.